### ISSUE TITLE
Wow

### ISSUE THREAD

**Issue by nollied** [danijar/dreamerv3#8]
No issue. just incredible work. Amazing code too, kinda messy and hard to interpret but that's because it's extremely general and efficient. 

Great work!!!!!!!

**Comment by DuaneNielsen** [comment]
Yeah.  I'll second that.  Best attempt at building a general AGI yet.  Model based RL is the future!

**Comment by zhoubin-me** [comment]
It can be better structured I guess.

**Comment by ZhangSanMo** [comment]
RL is the future！！！
